### DISCUSSION TITLE
rpi 5

### DISCUSSION THREAD

**Discussion by thaneye** [telekom-security/tpotce#1686] [General]
I am considering purchasing an rpi 5 (8GB RAM) for T-Pot HIVE install type Has anyone tried this?

**Comment by Ayaan7345** [comment]
It depend I would not recommend this because I'm assuming that you are going to put this in your home. Your public Ip might be at risk. A better option would be to get a vps they are pretty cheap I have one that is for $5 a month and they have cheaper options that start at 2.99 https://cloudfanatic.net
